SETUP/CHANGELOG.md

```diff
@@ -26,12 +26,20 @@ strings be picked up by the JS code.
 This is the last release to include the Greek Transliteration tool in the
 proofreading interface and the Greek Transliteration quiz.
 
+In the next release, project pages will require authentication and the
```

Comment:
This is assuming that we implement [this proposal mentioned in the forums](https://www.pgdp.net/phpBB3/viewtopic.php?t=84361). If not I will pull this entry before we release.